src/api/config/routes.rb

```diff
@@ -164,11 +164,10 @@ def self.matches?(request)
     end
 
     controller 'webui/project' do
-      get 'project/' => :index
-      get 'project/list_public' => :list_public
-      get 'project/list_all' => :list_all
+      get 'project/' => :index, as: 'projects'
+      get 'project/list_public' => :index
+      get 'project/list_all' => :index, show_all: true
```

Comment:
You speak in riddles to me;-)

I looked at /project/list_all and /project/list_public in the browser. And at least the spiderweb tests touch those I suppose. But that's somewhat random testing.